### PR TITLE
Added support for secondary (non-vowel) centering and Slavic languages.

### PR DESCRIPTION
**Pull request by henryso**
Addresses #495.

This branch is based on #492, so merging this merge changes for #492.  For the Slavic languages, I put in the letters from #489 and the suggestions from #491.

Please review.

### CODE REVIEW

**Comment by henryso** [comment]
Added explicit Hungarian support per #495.

File: tex/gregorio-vowels.dat [hunk 1]
```diff
@@ -55,7 +63,46 @@ prefix y Y;
...
+alias [Czech] to [Slavic];
+alias [slovak] to [Slovak];
+alias [Slovak] to [Slavic];
+
+language [Slavic];
+
+vowel aàáâąAÀÁÂĄ;
+vowel eèéêëěęEÈÉÊËĚĘ;
+vowel iìíîIÌÍÎ;
+vowel oòóôőOÒÓÔŐ;
```
Comment (eroux) [inline tex/gregorio-vowels.dat]: I don't think slavic languages have ő nor ű (but I'm not 100% sure)

**Comment by henryso** [comment]
@eroux Thanks, I forgot to remove the Hungarian vowels from the Slavic list. That's done now.

**Comment by jakubjelinek** [comment]
There are many other vowels that aren't used in Slavic languages, but I actually think it shouldn't hurt to list them some obvious vowels not present in the languages.  E.g. if in say a slavic hymn there are foreign words (latin, greek written in latin, etc.), there could be vowels the language does not have etc.
So I actually think with some comment it wouldn't hurt even to have the slavic vowels in the latin vowel set, etc. (after all, it right now has many french ones that aren't used in latin).

**Comment by henryso** [comment]
Right, this is actually the reason I left the Latin æ and œ as well as the accent and circumflex vowels in English, for loanwords and the likely use of some Latin in English text.  Should I add the Slavic vowels into the in-built Latin set?  I won't add the syllabic consonants.  People can always select the appropriate Slavic language should they need them.

**Comment by jakubjelinek** [comment]
If @eroux is not against it, I'm all for that.  Even without the secondary, with the u/U with ring above and e/E with caron in the set, the latin centering would be good enough for most of the slavic words, so e.g. if somebody forgots to request slavic centering, or if in a mostly latin song there are a few slavic words, the result will be usable.

**Comment by henryso** [comment]
The original request from @eroux in #489 was to add Slavic vowels to the Latin set, so I don't think he will be against it.  I'll add them.

**Comment by jakubjelinek** [comment]
Haven't read the centering code yet, what does it do for latin syllable "in"? Does it treat i as prefix even when it isn't followed by vowels?

**Comment by henryso** [comment]
Since the "i" in "in" is *not* followed by a vowel, then "i" is considered just a vowel (and not a prefix).  The same would be true for the "u" in "un".

**Comment by henryso** [comment]
I'm satisfied with how this works.  Anyone willing to review it?

**Comment by eroux** [comment]
ok for me

**Pull request merged**